index.html

```diff
@@ -5,523 +5,1137 @@
+              </svg>
+            </a>
+          </div>
+        </div>
       </div>
     </header>
 
     <!-- Main -->
-    <main>
+    <main class="main">
       <!-- Hero section -->
       <section class="hero">
-        <div class="hero-content">
-          <h1>Craft Your Success with Our <span>Digital</span> Assets</h1>
-          <p>
-            From our inspiring illustrations to dynamic UI/UX kits, elevate your
-            projects with our premium digital design resources.
-          </p>
-
-          <a href="./category.html" class="btn-shop">Shop Now</a>
-          <div>
-            <a href="#" class="btn-explore">
-              <img src="/" alt="play button" />
-              Watch About Us
-            </a>
+        <div class="container hero-section">
+          <div class="hero-content">
+            <h1 class="section-header hero-header">
+              <span class="header-line">Craft Your</span>
+              <span class="header-line">Success with </span>
+              Our
+              <span class="accent-header"> Digital</span> Assets
+            </h1>
+            <p class="hero-text">
+              from our inspiring illustrations to dynamic UI/UX kits, elevate
+              your projects with our premium digital design resources.
+            </p>
+            <div class="btns-header">
+              <a
+                href="./category.html"
+                class="btn-shop franklin-bold-16"
+                title="View Our Products"
+              >
+                Shop Now
+              </a>
+              <a href="#" class="btn-explore franklin-bold-16">
+                <div class="icon-play">
+                  <svg class="icon icon-play-img" width="16" height="16">
+                    <use xlink:href="./img/sprite.svg#icon-play"></use>
+                  </svg>
+                </div>
+                <span class="btn-explore-text">Watch About Us </span>
+              </a>
+            </div>
           </div>
-        </div>
 
-        <figure class="hero-images">
-          <img src="/" alt="UI/UX kits" />
-          <img src="/" alt="UI/UX kits" />
-          <img src="/" alt="UI/UX kits" />
-        </figure>
+          <figure class="hero-images">
+            <img
+              src="./img/img-hero/hero-photo1.webp"
+              alt="UI/UX kits - Smiling woman"
+              width="290"
+              height="164"
+              class="hero-img-woman"
+            />
+            <img
+              src="./img/img-hero/hero-photo3.webp"
+              alt="UI/UX kits - Website Template"
+              width="514"
+              height="300"
+              class="hero-img-website"
+            />
+            <img
+              src="./img/img-hero/hero-photo2.webp"
+              alt="UI/UX kits - Event Speaker on Laptop Screen"
+              width="280"
+              height="522"
+              class="hero-img-laptop"
+            />
+          </figure>
```

Comment:
I'm not sure, but maybe better way to save one image from figma instead of three. You can leave it as it is

Reply:
Maybe it would be easier to have a whole image. But for now i will leave it due to lack of time and deadlines.